frontend/views/components/CopyToClipboard.vue

```diff
@@ -0,0 +1,113 @@
+<template lang='pug'>
+.c-wrapper(@click='copyToClipboard')
+  input.c-invisible-input(
+    type='text'
+    ref='input'
+    :value='textToCopy'
+  )
+  slot
+  transition(name='fade')
```

Comment:
why do we need fade here?

Reply:
The animation was part of the design requirement. Margarida made a codepen example for how the animation should look like. :)